### PR TITLE
Low Temperature Radiant Water Systems Autosize Tube Length Correction

### PR DESCRIPTION
**Pull request by RKStrand**
This commit includes the fix, the units tests, and documentation
modifications to address this issue.

Pull request overview
---------------------
This work addresses GitHub Issue #6202.  In this issue, a user noted that when EnergyPlus autosizes the tube length for a water based radiant system it uses a tube spacing of 0.15m for all cases.  This is not necessary since the user enters a tube spacing in their input file as part of the construction definition.  This fix will now use the construction definition information and will only resort to using a tube spacing of 0.15m when information has not been entered or the spacing is not within realistic boundaries.

### Work Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [ ] At least one of the following appropriate labels must be added to this PR to be consumed into the changelog:
   - Defect: This pull request repairs a github defect issue #6202 

### Review Checklist
This will not be exhaustively relevant to every PR.
 - [ ] Code style (parentheses padding, variable names)
 - [ ] Functional code review (it has to work!)
 - [ ] If defect, results of running current develop vs this branch should exhibit the fix
 - [ ] CI status: all green or justified
 - [ ] Performance: CI Linux results include performance check -- verify this
 - [ ] Unit Test(s)
 - C++ checks:
   - [ ] Argument types
   - [ ] If any virtual classes, ensure virtual destructor included, other things
 - IDD changes:
   - [ ] Verify naming conventions and styles, memos and notes and defaults
   - [ ] Open windows IDF Editor with modified IDD to check for errors
   - [ ] If transition, add rules to spreadsheet
   - [ ] If transition, add transition source
   - [ ] If transition, update idfs
 - [ ] If new idf included, locally check the err file and other outputs
 - [ ] Documentation changes in place
 - [ ] Changed docs build successfully
 - [ ] ExpandObjects changes?
 - [ ] If output changes, including tabular output structure, add to output rules file for interfaces

### CODE REVIEW

File: doc/input-output-reference/src/overview/group-radiative-convective-units.tex
Comment (rraustad) [inline doc/input-output-reference/src/overview/group-radiative-convective-units.tex:1014]: incomplete sentence here and next also.

Note that if the user elects to autosize this field that a standard zone thermostat such as would be used ...

**Comment by rraustad** [comment]
CI shows 3 units test failures on i386 Windows 7 but not Win64 Windows 7? @Myoldmopar is this a problem?

EnergyPlusFixture.AirloopHVAC_VentilationRateProcedure
EnergyPlusFixture.FinAndOverhangCount
EnergyPlusFixture.TubularDaylightDiffuserCount

**Comment by Myoldmopar** [comment]
> is this a problem?

You bet.  But not a part of this PR.  Don't let it hold this up.  Once we get past this IDD freeze I'm going to dig my heels in on these random DElight, Fin, etc., issues and try to get them resolved.

**Review by rraustad — APPROVED** [review]
Approved with minor changes to docs as described.

**Comment by rraustad** [comment]
Merge contingent on cleaning up issues with CI testing.

File: doc/input-output-reference/src/overview/group-radiative-convective-units.tex
Comment (RKStrand) [reply doc/input-output-reference/src/overview/group-radiative-convective-units.tex:1014]: @rraustad Ok, thanks for catching that.  I have fixed this so hopefully it is grammatically and technically correct now.

@Myoldmopar @mjwitte  This should be ready to merge, but I am a little concerned that my GitHub desktop app did something weird again.  I updated from develop but then I got an error message about a file that I have changed (doc file) before I did the merge.  That seemed to confuse the app.  I committed the change and then did another update from develop and it showed more files that needed to be updated. I don't think there were any conflicts so hopefully it isn't going to be a problem.  If there is an issue, let me know.

Comment (rraustad) [reply doc/input-output-reference/src/overview/group-radiative-convective-units.tex:1014]: Well, I read it again and it sound OK. I got caught on the "such as" thinking it should have said ... such as ZoneHVAC:Thermostat ... what it should say is:

... thermostat, such as would be used for a forced air system, must ...

I wouldn't go to the trouble of changing that since I'm not sure it's actually needed.

Comment (rraustad) [reply doc/input-output-reference/src/overview/group-radiative-convective-units.tex:1014]: Oh, and I merged this 4 days ago. Didn't you get a notice?

Comment (RKStrand) [reply doc/input-output-reference/src/overview/group-radiative-convective-units.tex:1014]: @rraustad Oh rats...I remember that now...that explains the weirdness.  Sorry--my bad for poor multi-tasking.  Thanks for taking care of this and my apologies for not reading all of the messages fully.

@Myoldmopar @mjwitte Ignore anything going on in this branch this morning...it was already merged.  Sorry for any confusion this caused!

**Pull request merged**
